### PR TITLE
Load local data

### PR DESCRIPTION
**Pull request by leplatrem**
@n1k0 r?

### CODE REVIEW

File: scripts/components/App.js [hunk 1]
```diff
@@ -47,6 +47,13 @@ export class Store extends EventEmitter {
     this.collection = kinto.collection("items");
   }
 
+  load() {
```
Comment (n1k0) [inline scripts/components/App.js:50]: I'd have expected to see this called at least one time in non-test code

Comment (leplatrem) [inline scripts/components/App.js:50]: Updated

File: scripts/index.js [hunk 2]
```diff
@@ -8,4 +8,6 @@ const DEFAULT_SERVER_URL = "https://kinto.dev.mozaws.net/v1";
 
 const kinto = new Kinto({remote: DEFAULT_SERVER_URL});
 const store = new Store(kinto);
+store.load();
```
Comment (n1k0) [inline scripts/index.js]: How about calling this when App is mounted? as a bonus we could then write a test for it.

Comment (leplatrem) [inline scripts/index.js]: +1

File: scripts/components/App.js [hunk 3]
```diff
@@ -66,6 +74,10 @@ export default class App extends React.Component {
     this.props.store.on('change', this.setState.bind(this));
   }
 
+  componentDidMount() {
+    this.props.store.load();
```
Comment (n1k0) [inline scripts/components/App.js]: I think it can be moved to constructor with the same result.

File: test/index_test.js [hunk 4]
```diff
@@ -42,6 +43,10 @@ describe("App", () => {
       sinon.assert.calledWithExactly(store.collection.create, {label: "Hello, world"});
     });
 
+    it("loads items from store on mount", () => {
+      sinon.assert.called(store.load);
```
Comment (n1k0) [inline test/index_test.js]: nit: `calledOnce`

**Comment by n1k0** [comment]
r+wc

**Pull request merged**